<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>480</width>
    <height>516</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QTabWidget" name="tabWidget">
    <property name="enabled">
     <bool>true</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>80</y>
      <width>461</width>
      <height>391</height>
     </rect>
    </property>
    <property name="currentIndex">
     <number>1</number>
    </property>
    <widget class="QWidget" name="tab">
     <attribute name="title">
      <string>基本设置</string>
     </attribute>
     <widget class="QGroupBox" name="groupBox">
      <property name="geometry">
       <rect>
        <x>220</x>
        <y>10</y>
        <width>231</width>
        <height>171</height>
       </rect>
      </property>
      <property name="title">
       <string>自动采集</string>
      </property>
      <widget class="QCheckBox" name="checkBox">
       <property name="geometry">
        <rect>
         <x>20</x>
         <y>20</y>
         <width>41</width>
         <height>16</height>
        </rect>
       </property>
       <property name="text">
        <string>1线</string>
       </property>
      </widget>
      <widget class="QCheckBox" name="checkBox_2">
       <property name="geometry">
        <rect>
         <x>70</x>
         <y>20</y>
         <width>41</width>
         <height>16</height>
        </rect>
       </property>
       <property name="text">
        <string>2线</string>
       </property>
      </widget>
      <widget class="QCheckBox" name="checkBox_3">
       <property name="geometry">
        <rect>
         <x>120</x>
         <y>20</y>
         <width>41</width>
         <height>16</height>
        </rect>
       </property>
       <property name="text">
        <string>3线</string>
       </property>
      </widget>
      <widget class="QCheckBox" name="checkBox_4">
       <property name="geometry">
        <rect>
         <x>170</x>
         <y>20</y>
         <width>41</width>
         <height>16</height>
        </rect>
       </property>
       <property name="text">
        <string>4线</string>
       </property>
      </widget>
      <widget class="QCheckBox" name="checkBox_5">
       <property name="geometry">
        <rect>
         <x>20</x>
         <y>40</y>
         <width>41</width>
         <height>16</height>
        </rect>
       </property>
       <property name="text">
        <string>5线</string>
       </property>
      </widget>
      <widget class="QCheckBox" name="checkBox_6">
       <property name="geometry">
        <rect>
         <x>70</x>
         <y>40</y>
         <width>41</width>
         <height>16</height>
        </rect>
       </property>
       <property name="text">
        <string>6线</string>
       </property>
      </widget>
      <widget class="QCheckBox" name="checkBox_7">
       <property name="geometry">
        <rect>
         <x>120</x>
         <y>40</y>
         <width>41</width>
         <height>16</height>
        </rect>
       </property>
       <property name="text">
        <string>7线</string>
       </property>
      </widget>
      <widget class="QCheckBox" name="checkBox_8">
       <property name="geometry">
        <rect>
         <x>170</x>
         <y>40</y>
         <width>41</width>
         <height>16</height>
        </rect>
       </property>
       <property name="text">
        <string>8线</string>
       </property>
      </widget>
      <widget class="QLabel" name="label_9">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>70</y>
         <width>54</width>
         <height>12</height>
        </rect>
       </property>
       <property name="text">
        <string>登录人物：</string>
       </property>
      </widget>
      <widget class="QRadioButton" name="radioButton">
       <property name="geometry">
        <rect>
         <x>80</x>
         <y>70</y>
         <width>31</width>
         <height>16</height>
        </rect>
       </property>
       <property name="text">
        <string>左</string>
       </property>
      </widget>
      <widget class="QRadioButton" name="radioButton_2">
       <property name="geometry">
        <rect>
         <x>130</x>
         <y>70</y>
         <width>31</width>
         <height>16</height>
        </rect>
       </property>
       <property name="text">
        <string>中</string>
       </property>
      </widget>
      <widget class="QRadioButton" name="radioButton_3">
       <property name="geometry">
        <rect>
         <x>180</x>
         <y>70</y>
         <width>31</width>
         <height>16</height>
        </rect>
       </property>
       <property name="text">
        <string>右</string>
       </property>
      </widget>
     </widget>
     <widget class="QGroupBox" name="groupBox_2">
      <property name="geometry">
       <rect>
        <x>220</x>
        <y>190</y>
        <width>231</width>
        <height>171</height>
       </rect>
      </property>
      <property name="title">
       <string>变态功能</string>
      </property>
      <widget class="QCheckBox" name="checkBox_9">
       <property name="geometry">
        <rect>
         <x>20</x>
         <y>20</y>
         <width>91</width>
         <height>16</height>
        </rect>
       </property>
       <property name="text">
        <string>刷银鹤关守卫</string>
       </property>
      </widget>
      <widget class="QCheckBox" name="checkBox_10">
       <property name="geometry">
        <rect>
         <x>20</x>
         <y>40</y>
         <width>91</width>
         <height>16</height>
        </rect>
       </property>
       <property name="text">
        <string>刷金鹤关守卫</string>
       </property>
      </widget>
      <widget class="QCheckBox" name="checkBox_11">
       <property name="geometry">
        <rect>
         <x>20</x>
         <y>60</y>
         <width>61</width>
         <height>16</height>
        </rect>
       </property>
       <property name="text">
        <string>刷宝箱</string>
       </property>
      </widget>
     </widget>
     <widget class="QGroupBox" name="groupBox_4">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>10</y>
        <width>201</width>
        <height>351</height>
       </rect>
      </property>
      <property name="title">
       <string>基础属性</string>
      </property>
      <widget class="QLabel" name="label">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>20</y>
         <width>54</width>
         <height>12</height>
        </rect>
       </property>
       <property name="text">
        <string>人物：</string>
       </property>
      </widget>
      <widget class="QLabel" name="label_2">
       <property name="geometry">
        <rect>
         <x>40</x>
         <y>40</y>
         <width>54</width>
         <height>12</height>
        </rect>
       </property>
       <property name="text">
        <string>HP：</string>
       </property>
      </widget>
      <widget class="QLabel" name="label_3">
       <property name="geometry">
        <rect>
         <x>110</x>
         <y>40</y>
         <width>54</width>
         <height>12</height>
        </rect>
       </property>
       <property name="text">
        <string>MP：</string>
       </property>
      </widget>
      <widget class="QLabel" name="label_4">
       <property name="geometry">
        <rect>
         <x>40</x>
         <y>70</y>
         <width>54</width>
         <height>12</height>
        </rect>
       </property>
       <property name="text">
        <string>经验：</string>
       </property>
      </widget>
      <widget class="QLabel" name="label_5">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>100</y>
         <width>54</width>
         <height>12</height>
        </rect>
       </property>
       <property name="text">
        <string>宠物：</string>
       </property>
      </widget>
      <widget class="QLabel" name="label_6">
       <property name="geometry">
        <rect>
         <x>40</x>
         <y>130</y>
         <width>54</width>
         <height>12</height>
        </rect>
       </property>
       <property name="text">
        <string>HP：</string>
       </property>
      </widget>
      <widget class="QLabel" name="label_7">
       <property name="geometry">
        <rect>
         <x>40</x>
         <y>160</y>
         <width>54</width>
         <height>12</height>
        </rect>
       </property>
       <property name="text">
        <string>健康：</string>
       </property>
      </widget>
      <widget class="QLabel" name="label_8">
       <property name="geometry">
        <rect>
         <x>110</x>
         <y>130</y>
         <width>54</width>
         <height>12</height>
        </rect>
       </property>
       <property name="text">
        <string>MP：</string>
       </property>
      </widget>
     </widget>
    </widget>
    <widget class="QWidget" name="tab_3">
     <attribute name="title">
      <string>打怪设置</string>
     </attribute>
     <widget class="QGroupBox" name="groupBox_5">
      <property name="geometry">
       <rect>
        <x>220</x>
        <y>10</y>
        <width>231</width>
        <height>111</height>
       </rect>
      </property>
      <property name="title">
       <string>刷任务</string>
      </property>
      <widget class="QComboBox" name="taskListcomboBox">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>20</y>
         <width>211</width>
         <height>22</height>
        </rect>
       </property>
      </widget>
      <widget class="QPushButton" name="taskStartButton">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>80</y>
         <width>75</width>
         <height>23</height>
        </rect>
       </property>
       <property name="text">
        <string>开始</string>
       </property>
      </widget>
      <widget class="QPushButton" name="taskEndButton">
       <property name="geometry">
        <rect>
         <x>140</x>
         <y>80</y>
         <width>81</width>
         <height>23</height>
        </rect>
       </property>
       <property name="text">
        <string>结束</string>
       </property>
      </widget>
      <widget class="QLabel" name="label_10">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>50</y>
         <width>61</width>
         <height>16</height>
        </rect>
       </property>
       <property name="text">
        <string>当前执行：</string>
       </property>
      </widget>
      <widget class="QLineEdit" name="startTasklineEdit">
       <property name="enabled">
        <bool>false</bool>
       </property>
       <property name="geometry">
        <rect>
         <x>70</x>
         <y>50</y>
         <width>151</width>
         <height>20</height>
        </rect>
       </property>
      </widget>
     </widget>
    </widget>
   </widget>
   <widget class="QGroupBox" name="groupBox_3">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>20</y>
      <width>461</width>
      <height>51</height>
     </rect>
    </property>
    <property name="title">
     <string/>
    </property>
    <widget class="QPushButton" name="getGamePathButton">
     <property name="geometry">
      <rect>
       <x>290</x>
       <y>10</y>
       <width>75</width>
       <height>31</height>
      </rect>
     </property>
     <property name="text">
      <string>游戏目录</string>
     </property>
    </widget>
    <widget class="QPushButton" name="gameStartBt">
     <property name="geometry">
      <rect>
       <x>370</x>
       <y>10</y>
       <width>75</width>
       <height>31</height>
      </rect>
     </property>
     <property name="text">
      <string>启动游戏</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="gamePath">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>9</y>
       <width>271</width>
       <height>31</height>
      </rect>
     </property>
    </widget>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>480</width>
     <height>23</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
